packages/@react-aria/autocomplete/docs/useAutocomplete.mdx

```diff
@@ -0,0 +1,67 @@
+{/* Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/autocomplete';
+import {FunctionAPI, HeaderInfo, InterfaceType, TypeContext, TypeLink, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-aria/autocomplete/package.json';
+import statelyDocs from 'docs:@react-stately/autocomplete';
+import {InlineAlert, Content, Heading} from '@adobe/react-spectrum';
+
+---
+category: Pickers
+keywords: [autocomplete, autosuggest, typeahead, search, aria]
+after_version: 3.0.0
```

Comment:
this will mean the docs won't be deployed until 3.0 is released. We may need to wait until we update the hook docs to new collections anyway... Or at least this example would need to use the new implementation. I think the useAutocomplete hook depends on it.